### PR TITLE
Fix bug with mongo-db-collections (now note return always empty list)

### PR DESCRIPTION
**Pull request by neuromantic-js**
MongoDB changed and now procedure (mongo-db-collections db) always return empty list. I rewrite procedure and now all works.

### CODE REVIEW

File: db/mongodb/basic/driver.rkt [hunk 1]
```diff
@@ -177,18 +177,20 @@
...
-  (for/fold ([l empty])
-    ([c ans])
-    (define n (hash-ref c 'name))
-    (match (regexp-match name-rx n)
-      [(list _ name)
-       (if (regexp-match #rx"\\$" name)
-           l
-           (list* name l))]
-      [#f l])))
+  (let ([collections (mongo-db-collections db (list
```
Comment (jeapostrophe) [inline db/mongodb/basic/driver.rkt:180]: Is this a typo? Does it call itself?

Comment (jeapostrophe) [reply db/mongodb/basic/driver.rkt:180]: Should it be `mongo-db-execute-command!`?

Comment (neuromantic-js) [reply db/mongodb/basic/driver.rkt]: Yes, that is right. I did not notice how I made a mistake. In the current version of my branch, the master fixed it to a correct name.

**Comment by neuromantic-js** [comment]
Fix mistake with procedure name

**Comment by jeapostrophe** [comment]
Thank you very much!

**Pull request merged**